app/src/main/res/values/strings.xml

```diff
@@ -196,5 +196,6 @@
     <!--Format Strings-->
     <string name="double_and_String">%1$.2f%2$s</string>
     <string name="string_and_string">%1$s%2$s</string>
+    <string name="you_are_not_connected_to_the_internet">You are not connected to the internet!!</string>
```

Comment:
No need to add new string, please use already present string message ` <string name="no_internet_connection">No Internet Connection</string>`